lib/stanford/storage_object_validator.rb

```diff
@@ -0,0 +1,63 @@
+require 'moab'
+
+module Stanford
+  # druids are Stanford specific entities
+  class StorageObjectValidator < Moab::StorageObjectValidator
+    # error codes
+    EXPECTED_DATA_SUB_DIRS = ["content", "metadata"].freeze
```

Comment:
You shouldn't need to re-declare class constants -- you should be able to use them without redeclaring.